homeassistant/components/unifi/device_tracker.py

```diff
@@ -431,7 +431,12 @@ def extra_state_attributes(self):
         if self.device.state == 0:
             return {}
 
-        attributes = {}
+        attributes = {
+            CONNECTION_NETWORK_MAC: self.device.mac,
+            "manufacturer": ATTR_MANUFACTURER,
+            "model": self.device.model,
+            "sw_version": self.device.version,
```

Comment:
The other info is static and not part of our entity state model for device trackers so that's not allowed to be added to state attributes.

Reply:
@MartinHjelmare So, to clarify, only the MAC address is (currently) allowed as part of the state attributes?

My main use case is to display/use the software version and model (and manufacturer, but less important as I can hard-code that) in state template sensors and for display on the frontend dashboard.  Is there a way of accessing this data from the device tracker in this way?

Reply:
There's currently no way to do that. Templates only get access to state, ie not device info.

Reply:
Just reviewing the rest of the UniFi integration and it already appears to be providing static attributes into state for tracked client devices, as well as attributes not on the Device Tracker Entity properties list -- e.g. https://github.com/home-assistant/core/blob/922eec09098e0462b64bea91210a674512b8530a/homeassistant/components/unifi/device_tracker.py#L33-L56.  Since this was happening for client devices, providing relevant attributes for infrastructure seemed like it would fit.

If this isn't acceptable behaviour (as @Kane610 was mentioning), then where would a change for providing access to device info in the state be best discussed?  Should I start a discussion on https://github.com/home-assistant/architecture/?  I'm not wedded to the idea of putting this data into state attributes, but just somewhere it's possible to interact with on the frontend and available in templates / templated sensors.

Reply:
Yes, you can open an architecture discussion.

Reply:
Discussion opened at https://github.com/home-assistant/architecture/discussions/583

Reply:
@MartinHjelmare Do you have comments for the discussion at https://github.com/home-assistant/architecture/discussions/583 so we can make progress on this?

Reply:
Sorry, I don't have any comment. I don't think it's high priority at the moment to solve this.